### PR TITLE
fix: make docker on master (inexistent docker alpine version)

### PR DESCRIPTION
**Pull request by carloschida**
Currently when executing `make docker` on master, I get the following errors:

```shell
root@somewhere:~/kratos# make docker
DOCKER_BUILDKIT=1 docker build -f .docker/Dockerfile-build --build-arg=COMMIT=9097a60ede283fb2f6a912f2ab1ae1713d6598dd --build-arg=BUILD_DATE=2022-04-12T09:25:47Z -t oryd/kratos:latest .
[+] Building 2.0s (6/6) FINISHED                                                                                                                                                                                                                                         
 => [internal] load build definition from Dockerfile-build                                                                                                                                                                                                          0.0s
 => => transferring dockerfile: 38B                                                                                                                                                                                                                                 0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                   0.0s
 => => transferring context: 35B                                                                                                                                                                                                                                    0.0s
 => resolve image config for docker.io/docker/dockerfile:1-experimental                                                                                                                                                                                             0.7s
 => CACHED docker-image://docker.io/docker/dockerfile:1-experimental@sha256:600e5c62eedff338b3f7a0850beb7c05866e0ef27b2d2e8c02aa468e78496ff5                                                                                                                        0.0s
 => CANCELED [internal] load metadata for docker.io/library/alpine:3.15.4                                                                                                                                                                                           0.2s
 => ERROR [internal] load metadata for docker.io/library/golang:1.18-alpine3.15.4                                                                                                                                                                                   0.2s
------
 > [internal] load metadata for docker.io/library/golang:1.18-alpine3.15.4:
------
failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = docker.io/library/golang:1.18-alpine3.15.4: not found
```

This is consistent with the fact that the tag doesn't exist on [DockerHub](https://hub.docker.com/_/golang?tab=tags&page=1&name=1.18-alpine3.15).

The easy fix is to simply change the docker alpine version from 3.15.4 to 3.15.

## Related issue(s)
_None_

## Checklist

<!--
Put an `x` in the boxes that apply. You can also fill these out after creating the PR.

Please be aware that pull requests must have all boxes ticked in order to be merged.

If you're unsure about any of them, don't hesitate to ask. We're here to help!
-->

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: .docker/Dockerfile-build [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build go build -tags sqlite \
     -ldflags="-X 'github.com/ory/kratos/driver/config.Version=${VERSION}' -X 'github.com/ory/kratos/driver/config.Date=${BUILD_DATE}' -X 'github.com/ory/kratos/driver/config.Commit=${COMMIT}'" \
     -o /usr/bin/kratos
 
-FROM alpine:3.15.4
+FROM alpine:3.15
```
Comment (zepatrik) [inline .docker/Dockerfile-build:28]: Also this one
```suggestion
FROM alpine:3.15.4
```

File: .docker/Dockerfile-alpine [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.15.4
+FROM alpine:3.15
```
Comment (zepatrik) [inline .docker/Dockerfile-alpine:1]: Can you revert the plain alpine versions, as those exist and 3.15 has known CVEs detected by our scanners? https://snyk.io/test/docker/alpine%3A3.15.0
```suggestion
FROM alpine:3.15.4
```

**Review by zepatrik — CHANGES_REQUESTED** [review]
Sorry for breaking this, was completely my fault.

**Comment by carloschida** [comment]
I see the problem better now. Docker tags are really confusing sometimes.
Waiting for the pipeline.

**Pull request merged**